build.psm1

```diff
@@ -2813,6 +2821,8 @@ function Restore-PSModule
 
     log ("Name='{0}', Destination='{1}', Repository='{2}'" -f ($Name -join ','), $Destination, $RepositoryName)
 
+    # do not output progress
+    $ProgressPreference = "SilentlyContinue"
```

Comment:
nope - test tests are executed in a new scope, so it gets set back when the script exits